cmd/controller/main.go

```diff
@@ -179,8 +179,8 @@ func main() {
 	prometheusRegistry := prometheus.NewRegistry()
 
 	// before we acquire a lock we actually warm up caches mirroring the state of the API server
-	machineInformerFactory := machineinformers.NewSharedInformerFactory(machineClient, time.Second*30)
-	kubeInformerFactory := kubeinformers.NewSharedInformerFactory(kubeClient, time.Second*30)
+	machineInformerFactory := machineinformers.NewSharedInformerFactory(machineClient, time.Minute*15)
+	kubeInformerFactory := kubeinformers.NewSharedInformerFactory(kubeClient, time.Minute*15)
```

Comment:
oops

Reply:
"Oops" as in..? I changed it because the current interval is so short it will re-qeue Azure machines that are still blocking a goroutine. Also IMHO if a machine needs the resync we have a bug in the controller

Reply:
Opps as in, why did we ever had such a super short sync period?